examples/sparse_tensor.cu

```diff
@@ -195,8 +197,8 @@ int main([[maybe_unused]] int argc, [[maybe_unused]] char **argv) {
   auto doffsets = make_tensor<int>({3});
   dvals.SetVals({-1, -1, -1, -1, -1, 0, 4, 4, 4, 4, 4, 4, 0, 1, 1, 1, 1, 1});
   doffsets.SetVals({-1, 0, 1});
-  auto Adia = experimental::make_tensor_dia(dvals, doffsets, {6, 6});
-  print(Adia);
+  auto AdiaJ = experimental::make_tensor_dia<false>(dvals, doffsets, {6, 6});
```

Comment:
It may make more sense to use a tag type here instead so it's clear what they're doing. Something like:

```
struct INDEX_I{};
struct INDEX_J{};

make_tensor_dia<INDEX_I>(...);
```